Library/Homebrew/cmd/formulae.sh

```diff
@@ -16,9 +16,9 @@ homebrew-formulae() {
            -name vendor \
           \) \
          -prune -false -o -name '*\.rb' | \
-    sed -r -e 's/\.rb//g' \
-           -e 's_.*/Taps/(.*)/(home|linux)brew-_\1/_' \
-           -e 's|/Formula/|/|' \
+    sed -e 's/\.rb//g' \
+        -e 's_.*/Taps/\(.*\)/\(home\|linux\)brew-_\1/_' \
+        -e 's|/Formula/|/|' \
   )"
   local shortnames
   shortnames="$(echo "$formulae" | cut -d "/" -f 3)"
```

Comment:
Below this line: `echo -e "$formulae\n$shortnames" | \`

Reply:
This is being taken care of in #9366